tensorboard/plugins/pr_curve/summary.py

```diff
@@ -154,7 +154,7 @@ def op(
     summary_metadata = tf.SummaryMetadata(
         display_name=display_name if display_name is not None else tag,
         summary_description=description or '',
-        plugin_data=tf.SummaryMetadata.PluginData(plugin_name='pr_curve',
+        plugin_data=tf.SummaryMetadata.PluginData(plugin_name='pr_curves',
```

Comment:
Putting this here because I can't comment below: in the separate PR where you introduce these changes, on the return statement near line 179 please change `name=tag` to `name='pr_curves'`. The pattern followed by existing ops is to create a name-scope of the user-specified name, do any computations in there, and then create a summary with a _fixed_, plugin-specified name. We chose this pattern because it is also the pattern followed by TensorFlow ops.